security/sec-iride/src/main/java/org/geoserver/security/iride/IrideUserGroupService.java

```diff
@@ -0,0 +1,183 @@
+/*
+ *  GeoServer Security Provider plugin with which doing authentication and authorization operations using CSI-Piemonte IRIDE Service.
```

Comment:
Hi @afabiani, regarding the header comment the current format has been internally discussed and approved in CSI, and I would rather give it for granted as it is.

Nevertheless we will consider if it could be modified accordingly to your comment about its structure in future.